ext/standard/tests/array/compact_basic.phpt

```diff
@@ -19,12 +19,7 @@ $f="string";
 var_dump (compact(array("a", "b", "c", "d", "e", "f")));
 // simple parameter test
 var_dump (compact("a", "b", "c", "d", "e", "f"));
-var_dump (compact(array("keyval"=>"a", "b"=>"b", "c"=>1)));
-
-// cases which should not yield any output.
-var_dump (compact(array(10, 0.3, true, array(20), NULL)));
-var_dump (compact(10, 0.3, true, array(20), NULL));
-var_dump (compact(array("g")));
```

Comment:
Keeping the undefined variable case is probably a good idea, as in the other test is a bit more complicated.

Reply:
done